cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -744,6 +750,9 @@ impl ShowWithRRU for Inst {
                 ljustify("jmp".to_string()),
                 target.show_rru(mb_rru)
             ),
+            Inst::TrapIf { cc, trap_code, .. } => {
+                format!("j{} ; ud2 {} ;", cc.to_string(), trap_code)
```

Comment:
I think this needs to show the inverse of `trap_code` (viz, `trap_code ^ 1`) since what this currently displays is "skip the trap if `trap_code` is true".